entities/common.py

```diff
@@ -172,13 +197,14 @@ def __str__(self) -> str:
 
 
 class Polygon():
-    def __init__(self, uuid: str, layer: Layer, width: Width, fill: Fill, grab_area: GrabArea):
+    def __init__(self, uuid: str, layer: Layer, width: Width, fill: Fill,
+                 grab_area: GrabArea, vertices: Optional[List[Vertex]] = None):
```

Comment:
Could use `Iterable` instead of `List` here.

Reply:
True, I can change that :+1: